libraries/ESP8266WiFi/src/ESP8266WiFiSTA.h

```diff
@@ -46,6 +46,7 @@ class ESP8266WiFiSTAClass: public LwipIntf {
         //to detect Arduino arg order, and handle it correctly. Be aware that the Arduino default value handling doesn't
         //work here (see Arduino docs for gway/subnet defaults). In other words: at least 3 args must always be given.
         bool config(IPAddress local_ip, IPAddress gateway, IPAddress subnet, IPAddress dns1 = (uint32_t)0x00000000, IPAddress dns2 = (uint32_t)0x00000000);

```

Comment:
[If you can also fix this :\]](https://github.com/esp8266/Arduino/pull/9022#discussion_r1387252542)

Reply:
should I change it for `config` too to INADDR_ANY?